hommod_rest/services/utils.py

```diff
@@ -61,14 +63,10 @@ def extract_info(tar_path):
 
 
 def touch_file(path):
```

Comment:
You don't need a function for this. Just use `Path(path).touch()` where the function is called, and add a comment above it saying why you're using `touch`.